mm2src/coins/lightning/ln_platform.rs

```diff
@@ -422,17 +524,17 @@ impl FeeEstimator for Platform {
     fn get_est_sat_per_1000_weight(&self, confirmation_target: ConfirmationTarget) -> u32 {
         let platform_coin = &self.coin;
 
-        let default_fee = match confirmation_target {
-            ConfirmationTarget::Background => self.default_fees_and_confirmations.background.default_fee_per_kb,
-            ConfirmationTarget::Normal => self.default_fees_and_confirmations.normal.default_fee_per_kb,
-            ConfirmationTarget::HighPriority => self.default_fees_and_confirmations.high_priority.default_fee_per_kb,
+        let latest_fees = match confirmation_target {
+            ConfirmationTarget::Background => self.latest_fees.background.load(Ordering::Relaxed),
+            ConfirmationTarget::Normal => self.latest_fees.normal.load(Ordering::Relaxed),
+            ConfirmationTarget::HighPriority => self.latest_fees.high_priority.load(Ordering::Relaxed),
```

Comment:
Please consider using `Ordering::Acquire` memory order for these operations.

Reply:
Done